### DISCUSSION TITLE
I'm not sure if other models can be used

### DISCUSSION THREAD

**Discussion by dxandtom** [MadcowD/ell#215] [Q&A]
If I want to use the deepseek big model, this LLM call is slightly different from Chatgpt, in which case using ELL messages will be incorrect, can I change the source code to implement this function or can ELL not use other language models?

**Comment by dxandtom** [comment]
I found that I could redefine the client in line 240 of the complex.py function.

**Comment by MadcowD** [comment]
@dxandtom can you provide more context

**Comment by lingster** [comment]
I've succesfully used ell with ollama which provides access to a range of llms. see my reply here:
https://github.com/MadcowD/ell/discussions/368
